ocaml/xapi/vgpuops.ml

```diff
@@ -140,7 +140,7 @@ let add_vgpus_to_vm ~__context host vm vgpus vgpu_manual_setup =
       if not vgpu_manual_setup then
         let pgpu = allocate_vgpu_to_gpu ~__context vm host vgpu in
```

Comment:
For the record, Thomas and I have looked at this more closely, and concluded that there is no race, because all of this code runs with the global lock held from Message_forwarding.